packages/turf-line-segment/index.js

```diff
@@ -1,18 +1,13 @@
-var flatten = require('@turf/flatten');
-var meta = require('@turf/meta');
-var geomEach = meta.geomEach;
-var featureEach = meta.featureEach;
+var featureCollection = require('@turf/helpers').featureCollection;
+var flattenEach = require('@turf/meta').flattenEach;
 var getCoords = require('@turf/invariant').getCoords;
-var helpers = require('@turf/helpers');
-var feature = helpers.feature;
-var lineString = helpers.lineString;
-var featureCollection = helpers.featureCollection;
+var lineString = require('@turf/helpers').lineString;
 
 /**
  * Creates a {@link FeatureCollection} of 2-vertex {@link LineString} segments from a {@link LineString|(Multi)LineString} or {@link Polygon|(Multi)Polygon}.
  *
  * @name lineSegment
- * @param {Geometry|FeatureCollection|Feature<LineString|MultiLineString|MultiPolygon|Polygon>} geojson GeoJSON Polygon or LineString
+ * @param {GeometryCollection|Geometry|FeatureCollection|Feature<LineString|MultiLineString|MultiPolygon|Polygon>} geojson GeoJSON Polygon or LineString
```

Comment:
These JSDocs are becoming so long!... 😮 
I wonder if we could shorthand the Multi Geometries
```javascript
/**
 * @param {Geometry|FeatureCollection|Feature<(Multi)LineString|(Multi)Polygon>}
```